src/Microsoft.ApplicationInsights.AspNet/Config/TelemetryConfigurationSource.cs

```diff
@@ -0,0 +1,71 @@
+﻿namespace Microsoft.ApplicationInsights.AspNet.Config
```

Comment:
Can we follow the .NET guideline to avoid abbreviations and use the term `Configuration` instead of `Config`? It might also be a good idea to avoid creating this namespace at this point because it contains only two classes and violates the guideline & FxCop rule regarding namespace size.